flink-staging/flink-table/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
-			<version>${guava.version}</version>
+			<version>14.0.1</version>
```

Comment:
But then this could lead to problems with shading. Let's wait for @rmetzger on this, he probably knows this stuff best.